scss/_variables.scss

```diff
@@ -1001,6 +1001,7 @@ $form-floating-padding-x:         $input-padding-x !default;
 $form-floating-padding-y:         1rem !default;
 $form-floating-input-padding-t:   1.625rem !default;
 $form-floating-input-padding-b:   .625rem !default;
+$form-floating-label-height:      1.875em !default;
```

Comment:
Maybe we should use a calc function or some math here to get the `1.875em`?

Reply:
Actually, I've chosen this value arbitrarily, it _"feels right" to me. Open to any thoughts here!